galaxy_ng/app/access_control/access_policy.py

```diff
@@ -67,6 +67,10 @@ def has_rh_entitlements(self, request, view, permission):
         entitlement = entitlements.get(settings.RH_ENTITLEMENT_REQUIRED, {})
         return entitlement.get("is_entitled", False)
 
+    # if not defined, defaults to parent qs of None breaking Group Detail
+    def scope_queryset(self, request, view):
+        return view
```

Comment:
Actually the signature is `def scope_queryset(self, view, qs):`.

Reply:
Updated the parameter names, thanks.

The view does not have a `queryset_filtering_required_permission`.

Reply:
The parameters are (self, view, qs). Yes it does: https://github.com/ansible/galaxy_ng/blob/master/galaxy_ng/app/api/ui/viewsets/group.py#L34-L38. It inherits pulpcore's permission, but they overwrite the property to raise an error (which causes the `getattr` method to return `None`)

Reply:
Also looking at this:
https://github.com/pulp/pulpcore/blob/901499a3566a940516664ae6ca042455efee0115/pulpcore/app/viewsets/base.py#L353
I do not see how implementing this function as a noop is supposed to change a thing. Clearly i must be missing something.

Reply:
It's because this is the parent's `scope_queryset`: https://github.com/rsinger86/drf-access-policy/blob/master/rest_access_policy/access_policy.py#L67-L69

Reply:
Thank you. Now i understand. And now it's on us that we changed the function signature, because the request alone wasn't enough information for our usecase. :runner: